vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -323,6 +323,19 @@ testAuditDNotPresent() {
   echo "$test:Finish"
 }
 
+testMessageOfTheDayComponentsAzureLinux() {
+  os_sku=$1
+  if [[ "$os_sku" == "CBLMariner" || "$os_sku" == "AzureLinux" ]]; then
+    # this file controls how /etc/motd file is updated by dnf on azure linux nodes
+    # in CSE, if MotD is enabled, we modify that file to stop dnf from overwriting the user-specified message
+      if [[ ! -f /etc/default/automatic.conf ]]; then
```

Comment:
is this file ever not present on azlinux/mariner?

Reply:
That I don't know. But since we are relying on it to handle motd cases, I figured that we should have a test ensuring it is there and in case it disappears - investigate. Since those config files are coming directly from Azure Linux.

Reply:
Discussed offline, seems like this test is incorrect - those files are populated later on, so we will instead rely on cse checks and e2e tests